src/Backups/BackupCoordinationFileInfos.cpp

```diff
@@ -84,6 +84,10 @@ void BackupCoordinationFileInfos::prepare() const
         for (size_t i = 0; i != file_infos_for_all_hosts.size(); ++i)
         {
             auto & info = *(file_infos_for_all_hosts[i]);
+
+            if (!info.reference_target.empty())
+                continue;
```

Comment:
Plain backup implies that everything is stored without references. I mean a reference shouldn't be just skipped, a real file should be saved here instead. So for a plane backup the same file can be stored multiple times.